grafast/grafserv/src/servers/fastify/v4/index.ts

```diff
@@ -171,39 +176,48 @@ export class FastifyGrafserv extends GrafservBase {
     );
     */
 
-    const dynamicOptions = this.dynamicOptions;
-
-    app.route({
-      method:
-        this.dynamicOptions.graphqlOverGET ||
-        this.dynamicOptions.graphiqlOnGraphQLGET
-          ? ["GET", "POST"]
-          : ["POST"],
-      url: this.dynamicOptions.graphqlPath,
-      exposeHeadRoute: true,
-      bodyLimit: this.dynamicOptions.maxRequestLength,
-      handler: async (request, reply) => {
-        const digest = getDigest(request, reply);
-        const handlerResult = await this.graphqlHandler(
-          normalizeRequest(digest),
-          this.graphiqlHandler,
-        );
-        const result = await convertHandlerResultToResult(handlerResult);
-        return this.send(request, reply, result);
-      },
-      ...(this.resolvedPreset.grafserv?.websockets
-        ? {
-            wsHandler: makeHandler(makeGraphQLWSConfig(this)),
-          }
-        : null),
-    });
+    const {
+      graphiql,
+      graphiqlOnGraphQLGET,
+      graphiqlPath: url,
```

Comment:
You've collapsed `graphiqlPath` and `graphqlPath` into the same `url` setting which isn't quite right. Let's keep them named uniquely to avoid this kind of clash:

```suggestion
      graphqlPath,
      graphiqlPath,
```